### PR TITLE
LPS-181518 Update MDF Listing filter for partner view

### PR DESCRIPTION
**Pull request by Manda-vit**

### CODE REVIEW

**Comment by Manda-vit** [comment]
ci:test:stable

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/MDFRequestList.tsx [hunk 1]
```diff
@@ -170,73 +237,7 @@ const MDFRequestList = () => {
...
-													value: checkedItems,
-												},
-											}))
-										}
-									/>
-								),
-								name: 'Partner',
-							},
-						])}
+						menus={getDropDownFilterMenus(buildFilterFields())}
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/MDFRequestList.tsx:240]:
```suggestion
						menus={getDropDownFilterMenus(getFilters())}
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/MDFRequestList.tsx [hunk 2]
```diff
@@ -116,6 +116,73 @@ const MDFRequestList = () => {
 			);
 		}
 	};
+	const buildFilterFields = () => {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/MDFRequestList.tsx:119]:
```suggestion
	const getFilters = () => {
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by thaynam** [comment]
resent [3887](https://github.com/pt-liferay-solutions/liferay-portal/pull/3887)

**Pull request closed without merging**